features/aide/info.yaml

```diff
@@ -1,7 +1,7 @@
 # This subfeature can only be used with
-# its base feature: cis
-description: "CIS Subfeature: Aide"
+# its base feature: cis, fedramp
+description: "Aide"
 type: element
 features:
   depends:
-    - cis
+    - cis,fedramp
```

Comment:
This is problematic because once you enabled the aide feature, you must fulfill CIS and FedRAMP compliance otherwise the corresponding test will fail. However, it should be possible to choose either between CIS, FedRAMP or both.

We either handle this adjustments as flag feature that has no test or we add dedicated unit tests to them, that have nothing to do directly with fedramp or cis.